test/Tokenizer.t.sol

```diff
@@ -84,7 +92,7 @@ contract TokenizerTest is Test {
 
     function testUrl() public {
         vm.startPrank(originalOwner);
-        IPToken tokenContract = tokenizer.tokenizeIpnft(1, 100_000, "IPT", agreementCid, "");
+        IPToken tokenContract = oldTokenizer.tokenizeIpnft(1, 100_000, "IPT", agreementCid, "");
```

Comment:
haha, these tests oc should generally run on the "current" tokenizer (which doesn't really change). Just create a new test case that upgrades a newly deployed "old" tokenizer...